src/components/elements/tooltip/tooltip.styles.ts

```diff
@@ -0,0 +1,22 @@
+'use client';
+import * as RadixTooltip from '@radix-ui/react-tooltip';
+
+import type { StyledTooltipContentProps } from './tooltip.types';
+
+import { styled } from '@/styles';
+
+export const StyledTooltipContent = styled(RadixTooltip.Content)<StyledTooltipContentProps>`
+  background-color: ${({ theme }) => theme.colors.mediumGrey};
+  border-radius: 12px;
```

Comment:
According to the Figma, the radius should be 8px

<img width="788" alt="Screenshot 2024-02-26 at 17 13 55" src="https://github.com/bejamas/httptoolkit-website/assets/22168189/1fea6959-6980-47ae-a83f-b2061390ce87">